tox.ini

```diff
@@ -100,7 +96,8 @@ setenv =
     TWINE_USERNAME = {env:TWINE_USERNAME:__token__}
 commands =
     python -c "import path; path.Path('dist').rmtree_p()"
-    python -m pep517.build .
+    python -m build
```

Comment:
This change is technically beyond the intended scope of this PR, but it felt related and simple enough to sneak in.